GOVERNANCE.rst

```diff
@@ -384,27 +414,41 @@ If the proposed change cannot gather a 2/3 majority of the votes cast,
 the CC tries to resolve the deadlock.
 
 -  The CC will use consensus seeking.
--  If no consensus can be found within a month, the decision is made
-   through a simple majority vote (with tie breaking) among the CC
+-  If no consensus can be found within twenty working days excluding weekends
+   since the beginning of the stage-1 "reasonable time for consideration" period,
+   the decision is made through a simple majority vote (with tie breaking) among the CC
    members.
--  Any CC decision must be supported by an :ref:`steps`, which has been made public and discussed before the vote.
+-  If this vote is needed, the vote should take place within twenty-five working days after the beginning
+   of the stage-1 "reasonable time for consideration" period, during a regular, minuted CC meeting.
+-  Any proposal reaching stage 3 must be supported by an :ref:`steps`,
+   which has been made public at least 5 days, excluding week-ends, before the vote.
 
 .. _steps:
 
 sktime enhancement proposal
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For all decision of the CC, an sktime enhancement proposal (STEP) must have been made
-public and discussed before the vote.
+sktime enhancmenet proposals (STEPs) are required for:
 
-An sktime enhancement proposal is a consolidated document, with a concise
+* certain types of proposed changes, by default, see `decision making process <#Decision-making>`__
+* for all stage 3 decisions
+
+If a STEP is required by a vote, it must have been made public at least 5 working days (excluding week-ends) before that vote.
+
+A STEP is a consolidated document, with a concise
 problem statement, a clear description of the proposed solution and a
 comparison with alternative solutions, as outlined in our
 `template <https://github.com/sktime/enhancement-proposals/blob/master/TEMPLATE.md>`__.
 
-We collect and discuss proposals in sktime’s `repository for
+A complete STEP must always include at least a high-level design for the proposed change,
+not just a wishlist of features.
+
+Usually, we collect and discuss proposals in sktime’s `repository for
 enhancement-proposals <https://github.com/sktime/enhancement-proposals>`__.
 
+For smaller changes, such as punctual changes to the API or governance documents,
+the STEP can also be be part of an issue or pull request.
```

Comment:
for smaller changes we don't need a STEP I think, every PR should have some description of what's going on, but that's not what we consider a STEP I'd say 
```suggestion
For smaller changes, such as punctual changes to the API or governance documents,
the STEP can also be be part of an issue or pull request.
```

Reply:
then this would be in contradiction with the pre-existing requirements for decision making, which require a STEP for every change to the API or the governance document.

Reply:
Yes every "bigger" change

Reply:
it doesn't say that in the table (section decision making)

Reply:
Sure, but I wouldn't call PR descriptions STEPS, so I'd say for smaller changes no STEP is needed

Reply:
ok - suggestion is identical to the original, so I assume this is resolved.